customtypes/encodedstring/base64_input_type.go

```diff
@@ -0,0 +1,73 @@
+package encodedstring
```

Comment:
For now, the custom types are stored within this repo. If they need to be reused, we could create a separate go module containing all the custom types for our providers.

Reply:
We already have `shared-speakeasy` - please move to that repo

Reply:
Makes sense, will do!